pkg/mod/mod_init.go

```diff
@@ -0,0 +1,20 @@
+package mod
+
+import (
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+)
+
+const syslModuleName = "syslmodules"
+
+func SyslModInit(logger *logrus.Logger) error {
+	// makes the assumption that the CWD is not a go module since we hijack this command
+	out, err := exec.Command("go", "mod", "init", syslModuleName).CombinedOutput()
```

Comment:
Why are all modules named `syslmodules`?

Reply:
I didnt think the module name would matter, since it's a golang feature.

Reply:
It might cause problems later if we decide to start using it.